src/lang/en-US.json

```diff
@@ -561,6 +561,7 @@
   "story_item_purchase": "{player} purchased a {item} at {time}",
   "story_predicted_victory": "{players} predicted {team} would win",
   "story_predicted_victory_empty": "No one",
+  "story_networth_diff": "{percent}% / {gold} Diff",
```

Comment:
I don't think the percent part really needs to be localized, only the Diff part, but if this is easier then just keep it